pypesto/objective/history.py

```diff
@@ -14,6 +15,31 @@
 ResultType = Dict[str, Union[float, np.ndarray]]
 
 
+def trace_wrap(f):
+    """
+    Wrapper around trace getters that transforms input `ix` vectors to a valid
+    index list, and reduces for integer `ix` the output to a single value.
+    """
+    def wrapped_f(
+            self, ix: Union[Sequence[int], int, None] = None
+    ) -> Union[Sequence[Any], Any]:
```

Comment:
Can we be a bit more specific than `Any` here. Of course the function implements the most general case, but being so permissive may easily lead to false negatives in typechecking.